Dockerfile

```diff
@@ -37,7 +37,7 @@ ARG TARGETPLATFORM
 # See for details: https://github.com/hadolint/hadolint/wiki/DL4006
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-COPY scripts/apt /app/scripts/apt
+COPY scripts /app/scripts
```

Comment:
I don't like the idea of mixing apt and brew scripts, and putting them in the `/scripts` folder, so you have to copy the whole folder to the image, including brew scripts sub-folder (which is not going to be used there in any case)

Reply:
I expand the homebrew setup in README.

Reply:
This is good, however, my comment was more about the docker image. It is still not a big issue, as the brew-related files are copied to the build stages, not to the final image, but still, it looks a bit messy to me)

If you could briefly explain what is the benefit of this, it might be more clear to me)

Reply:
I don't have too many thought on this. I hope to keep the script as simple as possible. Copy all files under `scripts/` so the build script don't need to know the structure.

I don't have experience of writing a good development doc. I just write down what I did to set up my local environment.